content/docs/deployment/csmoperator/_index.md

```diff
@@ -15,17 +15,23 @@ Dell CSM Operator has been tested and qualified on Upstream Kubernetes and OpenS
 | -------------------------- | ------------------- |
 | 1.25, 1.26, 1.27           | 4.11, 4.12, 4.12 EUS |
 
+>NOTE:
+>- Authorization module is only supported on Kubernetes platforms.
```

Comment:
Why this note ?
If I use CSM Operator, this is to install components in Kubernetes.

Reply:
Hi @coulof, for installing the CSI-Driver with authorization module enabled via CSM-Operator, then with OCP platform its not supported yet. @sharmilarama please correct me if I missed anything.

Reply:
@coulof Authorization is not qualified on OCP platforms, and hence we do not claim support for it. The note was to indicate that.